contexts/did-v1.jsonld

```diff
@@ -116,6 +116,10 @@
     },
     "publicKeyBase58": "sec:publicKeyBase58",
     "publicKeyPem": "sec:publicKeyPem",
+    "publicKeyJwk": {
+      "@id": "sec:publicKeyJwk",
```

Comment:
publicKeyJwk is not described in https://w3id.org/security# (see https://github.com/w3c/did-core-registries/blob/ac4d80aa98e43837000a30fa18e4732f2a5d0e48/contexts/did-v1.jsonld#L8)

Reply:
You are 100% correct... this is a perfect example of what we are trying to fix.

The current security context updates are blocked by https://github.com/perma-id/w3id.org/pull/1638

The plan is to host all the security context documentation that did core depends on here at some point in the future, but perhaps we should consider fixing this now a blocker for this PR.